tests/examples/test_examples.py

```diff
@@ -93,10 +93,8 @@ def test_server_examples(server_example, example, report, bokeh_server):
         else:
             _get_pdiff(example)
 
-
-### {{{ THIS IS BROKEN and all examples are skipped in examples.yaml
 @pytest.mark.examples
-def test_notebook_examples(notebook_example, example, jupyter_notebook, report):
+def test_notebook_examples(notebook_example, example, jupyter_notebook, bokeh_server, report):
```

Comment:
Have to add the ``bokeh_server`` test fixture so that jupyter notebooks can load the resources.